frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.scss

```diff
@@ -1,43 +1,54 @@
 $border-color: var(--pf-global--BorderColor--light-100);
-$gutter: 1.8em;
 .odc-pipeline-vis-task {
   position: relative;
-  width: 13em;
-  height: 2em;
-  white-space: normal;
-  border: 1px solid $border-color;
-  border-radius: 5px;
-  background-color: var(--pf-global--BackgroundColor--200);
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: center;
+  width: 10em;
   margin-top: 1.5em;
-  padding: 0.5em;
+
   &__content {
     width: inherit;
+    height: 2.5em;
+    white-space: normal;
+    border: 1px solid $border-color;
+    border-radius: 20px;
+    color: black;
+    background-color: white;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12);
     display: flex;
-    position: relative;
-    top: 1px;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
```

Comment:
I think we want to left align the content
```suggestion

```

Justify Center:
<img width="153" alt="Screen Shot 2019-09-10 at 1 20 29 PM" src="https://user-images.githubusercontent.com/8126518/64635495-cfd35200-d3cd-11e9-90b0-0a1fc5d12f96.png">

Left-Aligned (omit style):
<img width="163" alt="Screen Shot 2019-09-10 at 1 20 36 PM" src="https://user-images.githubusercontent.com/8126518/64635522-deba0480-d3cd-11e9-9316-ac49302bfbe7.png">